netbout-web/src/test/java/com/netbout/email/EmAliasTest.java

```diff
@@ -49,6 +50,11 @@
 public final class EmAliasTest {
 
     /**
+     * Test email body.
+     */
+    private static final String EMAIL_BODY = "how are you?";
```

Comment:
@ndbroadbent we shouldn't to use  constants in unit tests to avoid duplicate string/numeric literals in test methods.
please check this [paper](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants)